merkle-tree/src/mmcs.rs

```diff
@@ -130,7 +130,10 @@ where
         // }
 
         // TODO: Disabled for now, CirclePcs sometimes passes a height that's off by 1 bit.
-        let max_height = dimensions.iter().map(|dim| dim.height).max().unwrap();
+        let Some(max_height) = dimensions.iter().map(|dim| dim.height).max() else {
+            // dimensions is empty
+            return Err(MerkleTreeError::WrongWidth);
```

Comment:
nit: I think I'd prefer a separate `EmptyBatch` just to make sure there's no confusion, but I'm also happy to make that tweak afterward

(I figure no code should really handle these error cases, in my mind they're akin to debug messages)